### ISSUE TITLE
Help Needed for Reproducing Paper Results with Released Code and Model

### ISSUE THREAD

**Issue by LL-E** [happychenpipi/EventSAM#14]
Thank you for sharing your excellent work! I recently evaluated the released pre-trained model (`rgbe_encoder.pth`) on the `test-normal` set of the `RGBE-SEG` dataset, but I noticed that the results were different from those reported in the paper. Additionally, I trained a model using the released code, but its performance also did not seem to match the reported results. Could you kindly provide guidance on properly training the model?  Your help would be greatly appreciated! Thank you!

| Metric | Paper Result | Released Model Result | My Result
|------------|---------------|-------------------|-------------|
| AP |   0.59 | 0.341               |     0.344     |
| AR  |  0.71  | 0.853               |     0.853     |
| mIoU  |  0.41 | 0.291               |     0.294     |
| aIoU  |  0.55 | 0.479               |      0.475    |
